tools/Resources.py

```diff
@@ -247,14 +247,19 @@ def get_latest_number_properties_multiple(target, token, uuids, propkey):
                 d = dict()
                 d['resourceId'] = resource['resourceId']
                 d['propkey'] = propkey
-                if 'values' in resource['property-contents']['property-content'][0]:
-                    d['data'] = resource['property-contents']['property-content'][0]['values'][0]
+                content = resource['property-contents']['property-content']
+                if content:
+                    if 'values' in content[0]:
+                        d['data'] = content[0]['values'][0]
+                    else:
+                        d['data'] = content[0]['data'][0]
                 else:
-                    d['data'] = resource['property-contents']['property-content'][0]['data'][0]
+                    print("skipping resource for get", str(propkey))
```

Comment:
Please add comment, we are getting a `None` reply here and therefore need to skip. Most likely due to a deleted object or some connection problem?